shared/test/test_assets.rb

```diff
@@ -182,6 +183,40 @@ def test_viewing_abusive_assets
     @api.delete_object(asset_name)
   end
 
+  def test_assets_copy_some
+    delete_all_assets('assets_test/1/2')
+    dest_channel_id = create_channel
+    src_api = FilesApiTestHelper.new(current_session, 'assets', @channel_id)
+    dest_api = FilesApiTestHelper.new(current_session, 'assets', dest_channel_id)
+
+    image_filename = 'çat.jpg'
+    image_body = 'stub-image-contents'
+
+    sound_filename = 'woof.mp3'
+    sound_body = 'stub-sound-contents'
+
+    _, image_filename = post_asset_file(src_api, image_filename, image_body, 'image/jpeg')
+    _, sound_filename = post_asset_file(src_api, sound_filename, sound_body, 'audio/mpeg')
+    src_api.patch_abuse(10)
```

Comment:
@pcardune - For context, If someone notices that an app is inappropriate or is violating our TOS, they can "report abuse," which effectively makes the app un-shareable.  If you go to the share link, it will just give you an error that says something to the effect of this app cannot be displayed because it violates our TOS.   

It makes sense to not let students copy assets over from projects that have been reported for abuse (at least not from projects that do not belong to them), since those projects should technically not be visible to them.  So we should check if the app's abuse score is >=10 (at least I think that's the check), and if it is, then show them an error in the dialog and say: "You cannot import from this project, as this project has been reported for violating Code.org's Terms of Service and cannot be shared with others."  (Link "Terms of Service" to code.org/tos).